tueplots/fontsizes.py

```diff
@@ -1,47 +1,53 @@
 """Fontsize settings."""
 
 
-def icml2022():
+def icml2022(*, default_smaller=1):
     r"""Font size for ICML 2022.
 
     Source: https://media.icml.cc/Conferences/ICML2022/Styles/example_paper.pdf
     """
     # ICML text size is 10, but captions are in size 9.
     # Therefore, we use base 9 instead of 10.
-    return _from_base(base=9)
+    return _from_base(base=9 - default_smaller)
 
 
-def neurips2021():
+def neurips2021(*, default_smaller=1):
     """Font size for Neurips 2021."""
-    return _from_base(base=10)
+    return _from_base(base=10 - default_smaller)
```

Comment:
The logic of calling `from_base` with a parameter `10 - default_smaller` instead of `9` is that 10 can be identified as neurips' fontsize, but if we called `from_base` with `base=9`, then the 9 might seem a little bit random (it might not be obvious that `from_base` is always called with one smaller than the journal's font-size). But with the named parameter, it is clearer, _and_ also provides access to changing it.